README.md

```diff
@@ -329,7 +324,7 @@ SOFTWARE.
 [maintenance-shield]: https://img.shields.io/maintenance/yes/2019.svg
 [patreon-shield]: https://www.frenck.nl/images/patreon.png
 [patreon]: https://www.patreon.com/frenck
-[pi-hole-sensor]: https://home-assistant.io/components/sensor.pi_hole/
```

Comment:
You missed a link on line 200

Reply:
You can use my changes in https://github.com/hassio-addons/addon-pi-hole/pull/108

Reply:
@hoppingmonk Thanks for your help!